Makefile

```diff
@@ -51,6 +51,8 @@ IMG ?= ghcr.io/qbarrand/oot-operator:main
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.23
 
+TEST ?= ./...
```

Comment:
Could we have that variable defined just before the `unit-test` target?

Reply:
Done.